app/views/dvr/challengeCasesDetails.scala.html

```diff
@@ -78,9 +78,36 @@
     }
 }
 
+@returnUrl = @{
+    if(isOwner)
+        s"${config.serviceUrl}${controllers.detailedvaluationrequest.routes.DvrController.myOrganisationRequestDetailValuationCheck(submissionId, assessmentRef, uarn).url}"
+    else
+        s"${config.serviceUrl}${controllers.detailedvaluationrequest.routes.DvrController.myClientsRequestDetailValuationCheck(submissionId, assessmentRef, uarn).url}"
+}
+
+@detailContent = {
+    <p>@messages("challengeRules.rules")</p>
+    <p>@messages("challengeRules.p1")</p>
+    <p>@messages("challengeRules.p2")</p>
+    <p>@messages("challengeRules.p3")</p>
+    <ul class="list bullets">
+        <li>@messages("challengeRules.rules.1")</li>
+        <li>@messages("challengeRules.rules.2")</li>
+        <li>@messages("challengeRules.rules.3")</li>
+    </ul>
+
+    <p>@Html(messages("challengeRules.rules.paper", config.businessRatesValuationFrontendUrl(s"property-link/valuations/startChallenge?backLinkUrl=$returnUrl")))</p>
+}
+
 
 <h2 class="govuk-heading-l">@messages("property.details.div.submitted.challenge.cases")</h2>
 
+@govukDetails(Details(
+    id = Some("whenYouMayChallengeDetails"),
```

Comment:
We usually format ids like `when-you-may-challenge-details`

Reply:
Thanks @DanSmith98, updated this & the existing challenge guidance component in checkCasesDetails.scala.html